src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -282,7 +282,19 @@ ucs_status_t uct_rdmacm_cm_ep_disconnect(uct_ep_h ep, unsigned flags)
     uct_rdmacm_cm_ep_t *cep = ucs_derived_of(ep, uct_rdmacm_cm_ep_t);
     char ip_port_str[UCS_SOCKADDR_STRING_LEN];
 
-    ucs_trace("rdma_disconnect on cm_id %p", cep->id);
+    ucs_trace("%s: rdma_disconnect on ep %p cm_id %p",
+              (cep->flags & UCT_RDMACM_CM_EP_ON_SERVER ? "server" : "client"),
```

Comment:
pls add helper function uct_rdmacm_ep_flags_str() to return string like "%c%c%c" representing ep flags to be used in all debug messages
also why need this debug message if already have one in the end of the function

Reply:
you mean to have every debug message print the strings representing all the flags that are set?
why 3 chars?

Reply:
yes. need more than 3 probably, 1 for each flag